circe-color-nicks.el

```diff
@@ -81,18 +81,22 @@ See `enable-circe-color-nicks'."
 recommended is 7:1, or at least 4.5:1 (7 stands for 7:1 here).
 Lower value allows higher color spread, but could lead to less
 readability."
+  :type 'number
```

Comment:
The other numbers are the same. They're all compared in `circe-nick-color-appropriate-p`. Although perhaps float values make a little sense there.